src/Details.js

```diff
@@ -10,7 +10,7 @@ class Details extends React.Component {
   state = { loading: true, showModal: false };
   componentDidMount() {
     pet
-      .animal(this.props.id)
+     .animal(parseInt(this.props.id))
```

Comment:
Don't forget the radix. You want to make sure that it gets parse to base 10.

```
parseInt(this.props.id, 10)
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

And actually in this case, I believe it would be more appropriate to use `Number()`

```
Number(this.props.id)
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number

Reply:
What you say sounds good. @btholt What do you think ?

In case you decide by Number I modify it